tests/integration/delete.bats

```diff
@@ -61,8 +61,8 @@ function teardown() {
 
   local subsystems="memory freezer"
 
-  for i in $(seq 1); do
-    runc run -d --console-socket $CONSOLE_SOCKET test_busybox
+  for _ in $(seq 1); do
```

Comment:
Why do we need this seq?

Reply:
Good question :) I wrote this code a month ago and I have no idea whatsoever :-\

My best guess is I wanted to repeat the test a few times (for some reason -- testing?), but yeah, it does not make any sense now.

Removing

Reply:
removed